docs/examples/DataSet/Dataset_Performance.ipynb

```diff
@@ -329,7 +329,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "As usual you can load the data using load_by_id but you will notice that the different storage methods\n",
+    "As usual you can load the data using ``load_by_id`` method but you will notice that the different storage methods\n",
```

Comment:
Missing article for `load_by_id`. Method->function.
(Maybe we need rewrite to make `load_by_spec` the default, @jenshnielsen what do you think?)